ignite/metrics/running_average.py

```diff
@@ -67,8 +70,9 @@ def compute(self):
         return self._value
 
     def attach(self, engine, name):
-        # restart average every epoch
-        engine.add_event_handler(Events.EPOCH_STARTED, self.started)
+        if self.epoch_bound:
+            # restart average every epoch
+            engine.add_event_handler(Events.EPOCH_STARTED, self.started)
```

Comment:
I think we need to call started at least once. So you can define the event for `self.started` handlers as 
```python
event = Events.STARTED if self.epoch_bound else Events.EPOCH_STARTED
engine.add_event_handler(event, self.started)
```
What do you think ?